plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/StmtConversionVisitor.kt

```diff
@@ -281,18 +302,51 @@ object StmtConversionVisitor : FirVisitor<Exp, StmtConversionContext<ResultTrack
         return UnitDomain.element
     }
 
+    @OptIn(DfaInternals::class)
```

Comment:
What is the DFA?  Why do we need the internals?

Reply:
Accessing the `symbol` property was requesting an `OptIn` for internals. I've pushed a fix addressing this!